### PR TITLE
Configuration changes and version update

### PR DESCRIPTION
**Pull request by Fermain**
This pull request is the culmination of testing in production scenarios. There are remaining issues to be resolved around how dependencies are managed but for the purposes of setting down a clear slate for the next team to work on the project - I am bringing these inline now.

You will note that this includes removing all the audio and text content to make for a more generic product. This process will continue, demo branches with content are fine.

### CODE REVIEW

File: packages/lms/src/lib/components/navigation/ProgrammeNav.svelte [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		const objectPath = path.replace('/content/', '');
 		pathArray = objectPath.split('/');
 		let currentPath = pathArray[pathArray.length - 1];
-		current = getCurrentPageEntityMeta(institutes || [], pathArray) || ({} as EntityMeta);
+		current = getCurrentPageEntityMeta(institutes || [], pathArray) || ({} as EntityMeta);
```
Comment (Fermain) [inline packages/lms/src/lib/components/navigation/ProgrammeNav.svelte:28]:
```suggestion
		current = getCurrentPageEntityMeta(institutes || [], pathArray) || ({} as EntityMeta);
```

File: packages/docs/package.json [hunk 2]
```diff
@@ -1,12 +1,15 @@
 {
   "name": "@mollify/docs",
-  "version": "1.0.0",
+  "version": "0.2.1",
```
Comment (pretzL) [inline packages/docs/package.json:3]: `docs` package version number is 0.2.1, everyone else's is 0.2.11. Is there a reason for this?

File: packages/lms/src/lib/components/navigation/EntityNav.svelte [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+			// border-left: 1rem solid;
+            // gap: 0.5rem;
+            // display: flex;
+            // flex-direction: column;
+
+			&:not(.open) {
+				display: none;
+			}
+		}
+	}
```
Comment (pretzL) [inline packages/lms/src/lib/components/navigation/EntityNav.svelte:69]: Remove commented out code?

File: packages/lms/src/lib/components/navigation/IconNav.svelte [hunk 4]
```diff
@@ -5,7 +5,7 @@
 </script>
 
 <AppRail width="w-[3rem] sm:w-20">
-	<AppRailTile name="tile-1" value={0}><i class="icon-f">home </i></AppRailTile>
+	<!-- <AppRailTile name="tile-1" value={0}><i class="icon-f">home </i></AppRailTile> -->
```
Comment (pretzL) [inline packages/lms/src/lib/components/navigation/IconNav.svelte:8]:
```suggestion
```

**Review by pretzL — APPROVED** [review]
Just some minor comments

File: packages/lms/src/lib/components/navigation/EntityNav.svelte (hunk 3 above)
Comment (Fermain) [reply packages/lms/src/lib/components/navigation/EntityNav.svelte:69]: Yeah I was going to do container queries here then... didn't

File: packages/docs/package.json (hunk 2 above)
Comment (pretzL) [reply packages/docs/package.json:3]: My OCD thanks you

**Pull request merged**